### ISSUE TITLE
Which configuration is used to reproduce the 74%?

### ISSUE THREAD

**Issue by AnthonyCAS** [phamquiluan/ResidualMaskingNetwork#7]
First of all thank you for this great contribution. I was trying to replicate you result with the Residual Masking Network, I used the fer2013 config file but with arch resmasking_dropout1 and channel 1.

Unfortunately I just got a 66% precision on the train set after almost 8 hours training(currently  I am working to speed up it using multiple TPUs).

I was wondering what configuration you use, also I notice there are some commented code in `resnet.py` like this:

`self.conv1 = nn.Conv2d(3, self.inplanes, kernel_size=7, stride=2, padding=3, bias=False)
#self.conv1 = nn.Conv2d(in_channels, self.inplanes, kernel_size=7, stride=2, padding=3, bias=False)
`

Since I am not doing transfer learning in fer2013 dataset, the channel size should be changed to 1 right?.

**Comment by phamquiluan** [comment]
I use image-net pretrained resnet34, so the `in_channels` is 3,

try to uncomment line 33 and line 34 

![image](https://user-images.githubusercontent.com/24642166/80986630-07aa7080-8e5b-11ea-9601-2f8b000e0880.png)

**Comment by AnthonyCAS** [comment]
I will try it. Thank you so much.
